### PR TITLE
Make sure all aliases exist (fix #408)

### LINKED ISSUE #408: networkudp alias is wrong
https://github.com/ManiacalLabs/BiblioPixel/blob/master/bibliopixel/project/alias_lists.py#L17
Should be `bibliopixel.drivers.network_udp.NetworkUDP`

### PR DESCRIPTION
**Pull request by rec**

### CODE REVIEW

File: bibliopixel/project/alias_lists.py [hunk 1]
```diff
@@ -7,18 +7,17 @@
 
 BUILTIN_ALIASES = {
     # drivers
-    'apa102': 'bibliopixel.drivers.API.APA102.APA102',
```
Comment (adammhaile) [inline bibliopixel/project/alias_lists.py]: Whaaaat?! How the hell did we miss that one! Well, other than the fact that I always use `spi` with `ledtype`

File: bibliopixel/project/alias_lists.py [hunk 2]
```diff
@@ -7,18 +7,17 @@
...
+    'apa102': 'bibliopixel.drivers.SPI.APA102.APA102',
+    'sk9822': 'bibliopixel.drivers.SPI.APA102.APA102',
     'dummy': 'bibliopixel.drivers.dummy_driver.Dummy',
     'hue': 'bibliopixel.drivers.hue.Hue',
-    'image': 'bibliopixel.drivers.image_sequence.ImageSequence',
-    'lpd8806': 'bibliopixel.drivers.API.LPD8806.LPD8806',
+    'lpd8806': 'bibliopixel.drivers.SPI.LPD8806.LPD8806',
     'network': 'bibliopixel.drivers.network.Network',
-    'network_udp': 'bibliopixel.drivers.network.NetworkUDP',
+    'network_udp': 'bibliopixel.drivers.network_udp.NetworkUDP',
```
Comment (adammhaile) [inline bibliopixel/project/alias_lists.py]: Clearly you didn't rebase in master as I already fixed this in v3.3.3

**Comment by rec** [comment]
Rebased.

We missed all of these because I did not write a test with this code.  

The "lack of test" indicates that I didn't think this through.

Though perhaps we didn't have the import machinery at that time.

I looked at the fact that none of these were tested and I guessed there would be two other mistakes and there were three when I actually tested it.  :-D  

(_And_ there was a file that can't be loaded if you don't have specific libraries.  I tweaked it to not be that way.)

**Comment by adammhaile** [comment]
Yeah, I think that went in before all that.
+1

**Pull request merged**